### PR TITLE
Added documentation for plum_lightpad component

### PR DESCRIPTION
**Pull request by heathbar**
**Description:**
Added documentation for the new plum_lightpad component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** TBD

### CODE REVIEW

File: source/_components/light.plum_lightpad.markdown [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+description: "Instructions how to integrate Plum Lightpads into Home Assistant."
+date: 2017-11-20 00:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: plum-lightpad.png
+ha_category: Light
+ha_iot_class: "Local Push"
+---
```
Comment (frenck) [inline source/_components/light.plum_lightpad.markdown]: Missing version tag

File: source/_components/light.plum_lightpad.markdown [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+To add a light switch, add the following section to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml
+light:
+  - platform: plum_lightpad
+    username: YOUR PLUM USERNAME
+    password: YOUR PLUM PASSWORD
+```
+
```
Comment (frenck) [inline source/_components/light.plum_lightpad.markdown]: I expected a list of configuration variables here. For more information check:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/light.plum_lightpad.markdown (hunk 1 above)
Comment (heathbar) [reply source/_components/light.plum_lightpad.markdown]: What is the next release? I'm guessing: 0.58.2?

Comment (frenck) [reply source/_components/light.plum_lightpad.markdown]: `0.59`, new features are never added in minor releases.

**Comment by heathbar** [comment]
Changes submitted

**Comment by frenck** [comment]
Closing this PR, since the parent PR was closed.

**Pull request closed without merging**